packages/f-checkout/src/store/checkout.module.js

```diff
@@ -84,6 +91,30 @@ export default {
             const response = await axios.post(url, data, config);
         },
 
+        /**
+         * Get the fulfilment details from the backend and update the state.
+         *
+         * @param {Object} commit - Automatically handled by Vuex to be able to commit mutations.
+         * @param {Object} payload - Parameter with the different configurations for the request.
+         */
+        getAvailableFulfilment: async ({ commit }, payload) => {
```

Comment:
If you're not using the `payload` variable anywhere in this action you could restructure the variables in place? 

```suggestion
        getAvailableFulfilment: async ({ commit }, { url, tenant, timeout }) => {
```